Library/ViewModels/ProjectPamphletContentViewModelTests.swift

```diff
@@ -6,30 +6,39 @@ import ReactiveSwift
 import XCTest
 
 final class ProjectPamphletContentViewModelTests: TestCase {
-  fileprivate let vm: ProjectPamphletContentViewModelType = ProjectPamphletContentViewModel()
+  fileprivate var vm: ProjectPamphletContentViewModelType!
```

Comment:
Is this change necessary? 🤔

Reply:
Nope! Sometimes it is for the scheduler to work correctly but in this case it wasn't. Thanks 👍